web/controllers/api/docs_controller.ex

```diff
@@ -112,19 +112,27 @@ defmodule HexWeb.API.DocsController do
     version         = to_string(release.version)
     unversioned_key = "docspage/#{package.name}"
     versioned_key   = "docspage/#{package.name}/#{release.version}"
+    pre_release     = length(release.version.pre) > 0
+
+    existing_paths = HexWeb.Store.list_docs_pages("#{name}/")
+    first_release  = length(existing_paths) == 0
```

Comment:
I also noticed there's a `packages.docs_updated_at` which it seems was supposed to be introduced for this feature, but I couldn't find where it's being set other than in the migration that introduced it. If that logic is missing, should I add it? Instead of doing a DB query we could just do: `first_release = package.docs_updated_at != nil` which would be nice.